packages/common/src/chains/garnet.ts

```diff
@@ -0,0 +1,32 @@
+import { chainConfig } from "viem/op-stack";
+import type { MUDChain } from "./types";
+
+const sourceId = 17000; // Holesky
+
+export const garnet = {
+  ...chainConfig,
+  id: 17069,
+  sourceId,
+  name: "Garnet Holesky",
+  testnet: true,
+  nativeCurrency: { name: "Ether", symbol: "ETH", decimals: 18 },
+  rpcUrls: {
+    default: {
+      http: ["https://rpc.garnetchain.com"],
+      webSocket: ["wss://rpc.garnetchain.com"],
+    },
+  },
+  blockExplorers: {
+    default: {
+      name: "Blockscout",
+      url: "https://explorer.garnetchain.com",
+    },
+  },
+  contracts: {
+    portal: {
+      [sourceId]: {
+        address: "0x57ee40586fbE286AfC75E67cb69511A6D9aF5909",
+      },
+    },
```

Comment:
do we have more contract addresses to expose?

[base](https://github.com/wevm/viem/blob/fdb6caabb2cf8c31144c7e427f38b9f0bd33e69f/src/chains/definitions/base.ts#L23-L46) and [base sepolia](https://github.com/wevm/viem/blob/fdb6caabb2cf8c31144c7e427f38b9f0bd33e69f/src/chains/definitions/baseSepolia.ts#L24-L47) for example have a bunch

Reply:
We can expose all the ones on Base too.

Reply:
What about the [MUD deployer](https://github.com/latticexyz/mud/blob/main/packages/cli/src/deploy/create2/deployment.json#L6)?

Reply:
> What about the [MUD deployer](https://github.com/latticexyz/mud/blob/main/packages/cli/src/deploy/create2/deployment.json#L6)?

since CREATE2 deployment itself is deterministic, I think folks just hardcode the address

there are no viem chains I could find with this address:
https://github.com/search?q=repo%3Awevm%2Fviem%200x4e59b44847b379578588920ca78fbf26c0b4956c&type=code
https://github.com/search?q=repo%3Awevm%2Fviem+create2&type=code

Reply:
should we add the OP stack contracts: https://github.com/wevm/viem/blob/fdb6caabb2cf8c31144c7e427f38b9f0bd33e69f/src/op-stack/contracts.ts (base has them too)

Reply:
we extend from this chain object